lib/MultiTopicsConsumerImpl.cc

```diff
@@ -897,6 +897,11 @@ void MultiTopicsConsumerImpl::afterSeek() {
     });
 }
 
+void MultiTopicsConsumerImpl::handleNoConsumersWhenSeek(const ResultCallback& callback) const {
+    LOG_ERROR(getName() << "There are no consumers when seek");
+    callback(ResultOperationNotSupported);
```

Comment:
`ResultOperationNotSupported` seems confusing to me. Maybe `ResultOperationIneffective ` is better.

Reply:
Adding a new error code brings an API change. Let's keep the same semantics as before that "ResultOperationNotSupported" indicates something like `IllegalStateException` in Java.